tools/pipelines/templates/include-policy-check.yml

```diff
@@ -28,14 +28,14 @@ stages:
 - stage: run_checks
   dependsOn: [] # Has no prereqs
   displayName: Policy checks
-  pool: Small
+  pool: Small-1ES
   jobs:
   - job:
     displayName: Run checks
     steps:
-    - template: include-use-node-version.yml
+    - template: /tools/pipelines/templates/1ES/include-use-node-version.yml@self
```

Comment:
Not sure the non-1ES templates should include the 1ES ones or run in the 1ES pools... unless this is already preparation to move everything to the new templates, but in that case maybe let's just update all the templates inline at their current location?